docs/eventing/README.md

```diff
@@ -3,6 +3,65 @@ Knative Eventing is a system that is designed to address a common need for cloud
 native development and provides composable primitives to enable late-binding
 event sources and event consumers.
 
+## Conceptual Overview
+
+### So you want to process events?
+
+Perfect, you’re in the right place. The resources in this section will guide you
+through some use cases and get you started in no time flat. There are different ways to go
+about handling event-driven architectures. In the resources in this section, we’ll share use
+cases and architecture examples ranging from straightforward to more complex.
+You can go through these resources in any order or only read the sections that you
+need. As a starting point, though, we’ll go over some terminology.
+
+#### What is event-driven
+
+Microservices, and functions, usually get their input from something else. That
+“something else” is the trigger for a microservice to take care of business. The
+piece of information that is sent across is called an event. The microservices
+in an event-driven architecture are constantly reacting to events and producing
+events themselves. As an example, think about your version control system. When
+you make changes to a project and want to merge it into the main branch, you
+need to create a Pull Request (PR). When you create that PR, it could trigger a
+bunch of other actions like code linting, quality tests, and update tickets.
+All those actions are started by events and the actions consume (act on) that
+data.
```

Comment:
I'd also remove this in favor or just providing a link to something like https://cloudevents.io/ in the very first paragraph - to me this explanation doesn't make it any clearer what events are, and the title is sort of misleading.